code/modules/martial_arts/krav_maga.dm

```diff
@@ -109,12 +109,12 @@
 		bonus_damage += 5
 		picked_hit_type = "stomps on"
 	D.apply_damage(bonus_damage, BRUTE)
-	if(picked_hit_type == "kicks" || picked_hit_type == "stomps")
+	if(picked_hit_type == "kicks" || picked_hit_type == "stomps on")
```

Comment:
I would also consider using is_horizontal here instead, as its better to have one source of truth, and it prevents weird stuff like this happening again if they change the picked_hit_type. However, if `D.apply_damage(bonus_damage, BRUTE)` can apply knockdown, this could be a problem. Perhaps the damage application could be moved below, or we could use a variable to store the boolean if they were horizontal when the damage was applied
```suggestion
	if(picked_hit_type == "kicks" || IS_HORIZONTAL(D))
```